resource_customizations/apps/DaemonSet/actions/discovery.lua

```diff
@@ -1,3 +1,4 @@
 local actions = {}
 actions["restart"] = {}
+actions["scale"] = {["defaultValue"] = tostring(obj.spec.replicas), ["hasParameters"] = true, ["errorMessage"] = "Enter any valid number", ["regexp"] = "^[0-9]*$"}
```

Comment:
Looks like the scale action is not relevant for Daemon Sets, since they operate on NodeSelectors.

Reply:
removed DaemonSet scale action